testutil/fixtures/load/frozen.go

```diff
@@ -82,28 +82,36 @@ func imageExists(client client.APIClient, name string) bool {
 }
 
 func loadFrozenImages(ctx context.Context, client client.APIClient) error {
-	tar, err := exec.LookPath("tar")
-	if err != nil {
-		return errors.Wrap(err, "could not find tar binary")
-	}
-	tarCmd := exec.Command(tar, "-cC", frozenImgDir, ".")
-	out, err := tarCmd.StdoutPipe()
-	if err != nil {
-		return errors.Wrap(err, "error getting stdout pipe for tar command")
-	}
+	frozenImages, _ := os.ReadDir(frozenImgDir)
+	for _, frozenImage := range frozenImages {
+		if frozenImage.IsDir() {
+			continue
+		}
+		fi, err := frozenImage.Info()
+		if err != nil {
+			return err
+		}
+		err = func(tarfile fs.FileInfo) error {
+			reader, err := os.OpenFile(path.Join(frozenImgDir, tarfile.Name()), os.O_RDONLY, 0644)
```

Comment:
Should probably use `filepath.Join`, not `path.Join` (I also thought "path" is for Linux, and "filepath" for "portable", but turns out `path` is nowadays advertised to only be used for URL(like) things.